### PR TITLE
Add one to many relation between opportunity and attachment

### PR DESCRIPTION
**Pull request by thomtrp**
[See issue ](https://github.com/orgs/twentyhq/projects/1/views/3?filterQuery=assignee%3Athomtrp+&visibleFields=%5B%22Title%22%2C%22Assignees%22%2C%22Status%22%2C46083682%2C%22Labels%22%2C46158959%2C%22Linked+pull+requests%22%2C%22Tracked+by%22%5D&pane=issue&itemId=52156209)

Capture d’écran 2024-02-06 à 18 21 46

### CODE REVIEW

**Review by Weiko — APPROVED** [review]
Lgtm

File: packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/attachment.object-metadata.ts [hunk 1]
```diff
@@ -80,4 +80,14 @@ export class AttachmentObjectMetadata extends BaseObjectMetadata {
...
+
+  @FieldMetadata({
+    type: FieldMetadataType.RELATION,
+    label: 'Opportunity',
+    description: 'Attachment opportunity',
+    icon: 'IconBuildingSkyscraper',
+    joinColumn: 'opportunityId',
+  })
+  @IsNullable()
+  opportunity: CompanyObjectMetadata;
```
Comment (charlesBochet) [inline packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/attachment.object-metadata.ts:92]: OpportunityObjectMetadata?

**Review by charlesBochet — CHANGES_REQUESTED** [review]

**Comment by charlesBochet** [comment]
Left a comment!

File: packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/attachment.object-metadata.ts (hunk 1 above)
Comment (Weiko) [reply packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/attachment.object-metadata.ts:92]: Fortunately this has no impact today but yes good catch! 😄

**Review by Weiko — APPROVED** [review]

**Review by charlesBochet — APPROVED** [review]

**Pull request merged**